### PR TITLE
 [prometheus-rabbitmq-exporter] update default rabbitmq secret key

### PR DESCRIPTION
**Pull request by vvinnich**
#### What this PR does / why we need it:
I think that bitnami rmq helm chart most popular. 
And they changed secret key for storing rabbitmq password from 'password' to 'rabbimq-password'

https://github.com/bitnami/charts/blob/master/bitnami/rabbitmq/templates/secrets.yaml#L12

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by monotek — CHANGES_REQUESTED** [review]
This would break the update for everyone already using an existing secret.
I also don't see the benefit of doing so.
Just because Bitnami does so is imho not an argument.

**Comment by vvinnich** [comment]
can I specify optional key with default "password" ?

File: charts/prometheus-rabbitmq-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ spec:
               valueFrom:
                 secretKeyRef:
                   name:  "{{ .Values.rabbitmq.existingPasswordSecret }}"
-                  key: password
+                  key: {{ .Values.rabbitmq.existingPasswordSecretKey | default "password" }}
```
Comment (monotek) [inline charts/prometheus-rabbitmq-exporter/templates/deployment.yaml:41]: Please use:
```
key: {{ .Values.rabbitmq.existingPasswordSecretKey }}
```

File: charts/prometheus-rabbitmq-exporter/values.yaml [hunk 2]
```diff
@@ -37,6 +37,7 @@ rabbitmq:
   password: guest
   # If existingPasswordSecret is set then password is ignored
   existingPasswordSecret: ~
+  # existingPasswordSecretKey:
```
Comment (monotek) [inline charts/prometheus-rabbitmq-exporter/values.yaml:40]: Please use:

```
existingPasswordSecretKey: password
```

**Review by monotek — CHANGES_REQUESTED** [review]

**Comment by monotek** [comment]
Please also sign your commits to get the dco check work.
See: https://github.com/prometheus-community/helm-charts/pull/778/checks?check_run_id=2149791210

**Comment by vvinnich** [comment]
commit already signed  and smth went wrong with DCO check

**Review by monotek — CHANGES_REQUESTED** [review]
I've updated the version to 0.7.0 as a new config option was added. 
Seems the dco check still fails. 

Please fix: https://github.com/prometheus-community/helm-charts/pull/778/checks?check_run_id=2150232973

Feel free to remove my commit and change the version by yourself.

File: charts/alertmanager/Chart.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@ icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/a
 sources:
   - https://github.com/prometheus/alertmanager
 type: application
-version: 0.7.0
+version: 0.7.1
```
Comment (monotek) [inline charts/alertmanager/Chart.yaml:9]: version needs to be 0.7.0

**Review by monotek — CHANGES_REQUESTED** [review]

**Review by monotek — APPROVED** [review]

**Pull request merged**